djangocms_alias/cms_toolbars.py

```diff
@@ -75,47 +76,49 @@ def add_alias_menu(self):
             position=1,
         )
 
-        alias_menu.add_modal_item(
-            _('View usage'),
-            url=admin_reverse(
-                USAGE_ALIAS_URL_NAME,
-                args=[self.toolbar.obj.alias_id],
-            ),
-        )
-
         can_change = self.request.user.has_perm(
             get_model_permission_codename(Alias, 'change'),
         )
         disabled = not can_change or not self.toolbar.edit_mode_active
         alias_menu.add_modal_item(
-            _('Edit alias details'),
+            _('Change alias settings'),
             url=admin_reverse(
-                'djangocms_alias_aliascontent_change',
-                args=[self.toolbar.obj.pk],
+                'djangocms_alias_alias_change',
+                args=[self.toolbar.obj.alias_id],
             ),
             disabled=disabled,
         )
         alias_menu.add_modal_item(
-            _('Change category'),
+            _('Rename alias'),
             url=admin_reverse(
-                'djangocms_alias_alias_change',
-                args=[self.toolbar.obj.alias_id],
+                'djangocms_alias_aliascontent_change',
+                args=[self.toolbar.obj.pk],
             ),
             disabled=disabled,
         )
         alias_menu.add_modal_item(
-            _('Delete Alias'),
+            _('View usage'),
             url=admin_reverse(
-                DELETE_ALIAS_URL_NAME,
-                args=(self.toolbar.obj.alias_id, ),
-            ),
-            on_close=admin_reverse(
-                LIST_ALIASES_URL_NAME,
-                args=(self.toolbar.obj.alias.category_id,),
+                USAGE_ALIAS_URL_NAME,
+                args=[self.toolbar.obj.alias_id],
             ),
-            disabled=disabled,
         )
 
+        # Only show deletion if versioning is not enabled
+        if not is_versioning_enabled():
+            alias_menu.add_modal_item(
+                _('Delete Alias'),
```

Comment:
The capitalisation here isn't consistent with the versioned menu option.

Reply:
This is where suggestions come into their own. 

```suggestion
                _('Delete alias'),
```